### PR TITLE
UI

### PR DESCRIPTION
**Pull request by Sunaries**
UI additions. Some I do not know without context and respective menu. 

Merry Christmas @imcgames

### CODE REVIEW

File: UI.tsv [hunk 1]
```diff
@@ -1,8 +1,8 @@
-UI_20150317_000001	{@st41b}Learn Attributes{nl}from the Masters of each class.{/}
+UI_20150317_000001	{@st41b}Learn Attributes{nl}from the representative masters of each class.{/}
```
Comment (ttgmichael) [inline UI.tsv]: from respective masters of each class?

**Comment by Sunaries** [comment]
Good to merge.

File: UI.tsv [hunk 2]
```diff
@@ -1828,153 +1828,153 @@ UI_20151022_001827	{#000000}{s18}The server is currently full. Queuing for entry
...
+UI_20151224_001898	%s (Ch %s)
+UI_20151224_001899	Number of Fragments
+UI_20151224_001900	Competitive Mission
 UI_20151224_001901
-UI_20151224_001902
-UI_20151224_001903
-UI_20151224_001904
-UI_20151224_001905
+UI_20151224_001902	{@st41b}Enter
+UI_20151224_001903	{@st59b}Quests{nl}{@st59s} - When you are on a new quest, it will be added to the quests window.{nl} - A main quest is displayed in yellow color, while a subquest is displayed in blue.{nl} - The arrow on your character's foot leads to the location of your quest{nl} - A 'Return' icon is displayed in the quests window when a quest is completed{nl}  (Teleports you to the Quest NPC using the Backspace key or a mouse-click){nl} - Add or remove quests in the quests window by ticking or unticking the box in front of the quest name{/}
```
Comment (mamoruk) [inline UI.tsv]: UI_20151224_001903  Life

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Happy Holidays @Sunaries :+1:

**Pull request merged**